keyboards/ferris/keymaps/madhatter/rules.mk

```diff
@@ -0,0 +1,2 @@
+BOOTMAGIC_ENABLE = yes
```

Comment:
This should be set to full or lite, ideally. 

And "lite" is the preferred option, as it's less prone to weird issues.

Reply:
done

Reply:
made it lite